scripts/configure-multinode.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+set -e
+
+WORKER_NODE_HOSTNAME=${WORKER_NODE_HOSTNAME:-"worker-node"}
+: "${WORKER_NODE_IP:=$(getent hosts "${WORKER_NODE_HOSTNAME}" | awk '{print $1}')}"
+KUBELET_DIR=${KUBELET_DIR:-"/home/microshift/kubelet"}
```

Comment:
```suggestion
KUBELET_DIR=${KUBELET_DIR:-"~/kubelet"}
```

Reply:
Would `$HOME` instead of `~` work better?